resources/stsci.edu/schemas/spline1d-1.0.0.yaml

```diff
@@ -0,0 +1,61 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/transform/spline1d-1.0.0"
+tag: "tag:stsci.edu:asdf/transform/spline1d-1.0.0"
```

Comment:
We should get in the habit of not including this validator, since it prevents others (and ourselves) from building off of our schema.